fea-rs/src/compile/validate.rs

```diff
@@ -751,6 +751,11 @@ impl<'a> ValidationCtx<'a> {
             );
         }
         let mut kind = None;
+
+        // tracking lookupflag declarations:
+        // you can set a lookupflag before seeing any rules,
+        // after the last rule, and not anywhere else.
+        let mut has_reset_lookup_flag = None;
```

Comment:
has_ to me strongly implies a bool and yet we set None with no type. I have to go read the rest of the file to figure out what this is, if it was Some(thing) what would thing be?

Perhaps a comment and/or type to indicate what values this can assume?